--- config.mk	2022-06-27 09:41:34.255768393 +0100
+++ config.mk	2022-06-27 09:43:07.665767223 +0100
@@ -18,9 +18,9 @@
 FREETYPELIBS = -lfontconfig -lXft
 FREETYPEINC = /usr/include/freetype2
 # OpenBSD (uncomment)
-#FREETYPEINC = ${X11INC}/freetype2
-#KVMLIB = -lkvm
-#MANPREFIX = ${PREFIX}/man
+FREETYPEINC = ${X11INC}/freetype2
+KVMLIB = -lkvm
+MANPREFIX = ${PREFIX}/man

 # includes and libs
 INCS = -I${X11INC} -I${FREETYPEINC}
